DESCRIPTION

```diff
@@ -58,6 +58,7 @@ Suggests:
     knitr,
     lintr,
     markdown,
+    potools,
```

Comment:
potools is only used in the development process (the same way we use, e.g., roxygen2 or pkgdown) and does not need to be added as a dependency